docs/index.rst

```diff
@@ -30,7 +30,10 @@ In addition, there are several extension packages which can be installed separat
 
 The extension packages can be found in :scm_web:`ext/ directory of the repository <ext/>`.
 
-In addition, third party exporters are available:
+Extensions
+----------
+
+OpenTelemetry supports integration with the following third party exporters:
 
 * `Azure Monitor <https://github.com/microsoft/opentelemetry-exporters-python/tree/master/azure_monitor>`_
```

Comment:
I would suggest pointing users to https://opentelemetry.io/registry to find more exporters

Reply:
Done!